src/libs/blueprint/conduit_blueprint_mesh.cpp

```diff
@@ -1043,6 +1043,101 @@ convert_coordset_to_explicit(const std::string &base_type,
     }
 }
 
+//-------------------------------------------------------------------------
+void
+copy_node_rename_first_child(const conduit::Node & from,
+                             conduit::Node & to,
+                             const std::string & key,
+                             const std::string & new_name,
+                             const std::string & new_sibling)
+{
+    to[key].set(from[key]);
+    conduit::Node & n = to[key];
+    const std::string from_name = n.child(0).name();
+    n.rename_child(from_name, new_name);
+    n[new_sibling].set(n[new_name]);
```

Comment:
Despite the name - I am having a hard time parsing this. 

Do we need to pass from & to and the child key?
Why not just pass those nodes like:
`copy_node_rename_first_child(src[key],dest[key], new_name, new_sibling)`

I also don't understand the context of `new_sibling`? 

I see the rename, but then `n[new_sibling]` is a copy of the first child after it was renamed?